iaso/api/org_unit_change_request_configurations/filters.py

```diff
@@ -8,6 +8,9 @@
 
 class OrgUnitChangeRequestConfigurationListFilter(django_filters.rest_framework.FilterSet):
     project_id = django_filters.NumberFilter(field_name="project", label=_("Project ID"))
+    type = django_filters.ChoiceFilter(
+        field_name="type", label=_("Type"), choices=OrgUnitChangeRequestConfiguration.Type.choices
+    )
```

Comment:
`django_filters` is smart enough to infer this line only by adding `"type"` to the `fields` list in the inner `class Meta`:

```python
    class Meta:
        model = OrgUnitChangeRequestConfiguration
        fields = ["type"]
```

This can save a bunch of typing :)